neo/NeoSystem.cs

```diff
@@ -40,10 +41,18 @@ public NeoSystem(Store store)
         public void Dispose()
         {
             RpcServer?.Dispose();
-            ActorSystem.Stop(LocalNode);
+            EnsureStoped(LocalNode);
```

Comment:
Do you think `LocalNode` is all that needs to be stopped and waited on for safe shutdown?

Reply:
We can use `EnsureStoped()` to sort the order of stops. Currently, it seems that only `LocalNode` needs to be stopped first.